docs/tooling/cli-guides/setup-a-devnet.md

```diff
@@ -0,0 +1,336 @@
+---
+tags: [Tooling, Avalanche-CLI]
+description: This page demonstrates how to setup a Devnet of cloud-based validators using the Avalanche-CLI.
+pagination_label: Setup a Devnet of Cloud-Based validators using Avalanche-CLI
+sidebar_label: Setup a Devnet
+sidebar_position: 8
+---
+
+# Setup a Devnet Using Avalanche-CLI
+
+This page demonstrates how to setup a Devnet of cloud-based validators using Avalanche-CLI, 
+and deploy a VM into it.
+
+Devnets (Developer Networks) are isolated avalanche networks deployed on the cloud. Similar to local networks
+in terms of configuration and usage but installed on remote nodes.
+
+Think of DevNets as being an intermediate step in the developer testing process after local network and before Fuji network.
+
+:::warning
+
+ALPHA WARNING: This command is currently in experimental mode. Proceed at your own risk.
+
+:::
+
+## Prerequisites
+
+Before we begin, you will need to have:
+
+- Created an AWS account and have an updated AWS `credentials` file in home directory with [default] profile
+- Install [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli)
+- [Created a Subnet configuration `<SubnetName>`](/build/subnet/deploy/fuji-testnet-subnet.md#create-an-evm-subnet) based on SubnetEVM.
+
+Note: the tutorial is based in AWS, but Devnets can also be created and operated in other supported
+cloud providers, such as GCP.
+
```

Comment:
This is an amazing and super useful tutorial ty for that! Just please add a "Choose a deployment method" section including "Single command" and "Step by step" options. This to avoid developers going though the whole process just to later realize there is a simpler way.

Reply:
Thanks for your kind comment! I addressed!